cmd/server/assets/realmadmin/stats.html

```diff
@@ -34,6 +34,8 @@ <h1>Realm stats</h1>
     </div>
 
     {{template "realmadmin/_stats_tokens" .}}
+
+    {{template "realmadmin/_stats_keyserver" .}}
```

Comment:
iirc, this needs to be a realm-level configurable. We should only display this if the realm enabled it.

Reply:
the realm-enablement currently guards the stats-puller from storing data in the first place. Do we need it here too?

Reply:
added an explicit guard